registry-analysis/Readme.md

```diff
@@ -43,14 +44,15 @@ To choose a mechanism and the rules, we document our expectations below:
     - It breaks interoperability otherwise
     - We should avoid confusing users, i.e. which binding should I use?
     - It would complicate implementations
-  - TODO: Clarify what happens when two ecosystems like OCF and LwM2M both use CoAP binding. The initial thinking is to register them as separate entries but clarify what they use from CoAP binding. A layered registry can be thought of as language tags with `en` extending to `en-us` and `en-uk` where the tags and entries are different but the association to `en` is expressed in the id.
+  - TODO: Clarify what happens when two ecosystems like OCF and LwM2M both use CoAP binding. The initial thinking is to register them as separate entries but clarify what they use from CoAP binding. A layered registry can be thought of as language tags with `en` extending to `en-us` and `en-uk` where the tags and entries are different but the association to `en` is expressed in the id. In our case, this would be `coap-ocf` and `coap-lwm2m`. The higher-level binding SHOULD NOT override or conflict the lower-level binding and this should be verified by the custodian. E.g. `cov:method` in CoAP binding should not be turned into `cov:operation` in the higher-level binding. The namespace (prefix and its values) defined in a binding CANNOT be redefined in any other binding.
```

Comment:
```suggestion
  - TODO: Clarify what happens when two ecosystems like OCF and LwM2M both use CoAP binding. The initial thinking is to register them as separate entries but clarify what they use from CoAP binding. A layered registry can be thought of as similar to language tags with `en` extending to `en-us` and `en-uk`, where the tags and entries are different but the association with `en` is expressed in the id. In our case, this would be `coap-ocf` and `coap-lwm2m`. A higher-level binding SHOULD NOT override or conflict with a lower-level binding, and this should be verified by the custodian, e.g., `cov:method` in CoAP binding should not be turned into `cov:operation` in the higher-level binding. The namespace (prefix and its values) defined in a binding CANNOT be redefined in any other binding.
```